src/processor/mapper/plan_mapper.cpp

```diff
@@ -208,6 +211,29 @@ std::unique_ptr<PhysicalOperator> PlanMapper::appendResultCollectorIfNotCopy(
     return lastOperator;
 }
 
+// TODO: re-visit before merge
+// std::unique_ptr<PhysicalOperator> PlanMapper::appendResultCollectorIfNotCopy(
```

Comment:
Seems you have figured out the way to return a column for CopyTo. So you can just remove this commented code.